src/fuchsia/mod.rs

```diff
@@ -3250,6 +3250,60 @@ f! {
         dev |= (minor & 0xffffff00) << 12;
         dev
     }
+
+    pub fn CMSG_DATA(cmsg: *const cmsghdr) -> *mut c_uchar {
+        cmsg.offset(1) as *mut c_uchar
+    }
+
+    pub fn CMSG_NXTHDR(mhdr: *const msghdr, cmsg: *const cmsghdr) ->
+        *mut cmsghdr {
```

Comment:
Hm this indent is unusual, should be:
```suggestion
    pub fn CMSG_NXTHDR(mhdr: *const msghdr, cmsg: *const cmsghdr)
        -> *mut cmsghdr
    {
```

Reply:
Done